### PR TITLE
Add API to increase log_count

### PR DESCRIPTION
**Pull request by opan**

### CODE REVIEW

**Comment by opan** [comment]
@tagnotfound Should we test controller too? I don't see any spec for controller yet

**Comment by giosakti** [comment]
I think we should have unit test for controller @opan, me and tara encounter some bugs because it's not tested yet.

after this PR is merged, can you take a look into that?

**Comment by opan** [comment]
@giosakti Sure, will be working on it.

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ group :development, :test do
   gem 'rspec-rails', '<= 3.7.2'
   gem 'timecop', '<= 0.9.1'
   gem 'webmock', '<= 3.4.1'
+  gem 'rails-controller-testing'
```
Comment (tagnotfound) [inline Gemfile]: Why is this needed? Isn't rsepc-rails enough?

Comment (opan) [reply Gemfile]: @tagnotfound I need to test the controller is render the right template. https://github.com/rails/rails-controller-testing
